composer.json

```diff
@@ -55,7 +55,7 @@
     "extra": {
         "typo3/cms": {
             "cms-package-dir": "{$vendor-dir}/typo3/cms",
-            "extension-key": "skilldisplay",
+            "extension-key": "skilldisplay_content",
```

Comment:
Also needs to be changed in line 52, where it is symlinked for functional tests.
We still don't have functionals, but once an functional test is added, the extension name would be different.

Reply:
Right and actually that's even outdated, will replace with: `TYPO3\\TestingFramework\\Composer\\ExtensionTestEnvironment::prepare`